### DISCUSSION TITLE
Embed application hosted on SWA in MS Teams

### DISCUSSION THREAD

**Discussion by adamtay** [Azure/static-web-apps#1028] [Q&A]
Hey there!

I have a React Application hosted on SWA, and would like to embed it as a Personal App on MS Teams.

The problem is that the React App redirects to the Azure AD login screen, which cannot be embedded within the Personal App, given that it is essentially running under an iFrame.

How could I best go about embedding the React App, meanwhile ideally being able to SSO from the MS Teams credentials and not prompt the user having to re-authenticate?

Thanks

**Comment by MWG-Logan** [comment]
I'm curious about a solid solution here as well.
